pkg/collector/process.go

```diff
@@ -266,12 +270,13 @@ func getMessageLength(msgBuffer *bytes.Buffer) (int, error) {
 // (encoding reference: https://tools.ietf.org/html/rfc7011#appendix-A.5)
 func getFieldLength(dataBuffer *bytes.Buffer) int {
 	lengthBuff := dataBuffer.Next(1)
-	var lengthVal uint16
-	util.Decode(bytes.NewBuffer(lengthBuff), &lengthVal)
-	if lengthVal < 255 { // string length is less than 255
-		return int(lengthVal)
+	var lengthVal1 uint8
+	util.Decode(bytes.NewBuffer(lengthBuff), &lengthVal1)
+	if lengthVal1 < 255 { // string length is less than 255
```

Comment:
was there an issue using one variable?

Reply:
I see the difference. lengthOneByte and lengthTwoBytes are better names. What do you think?

Reply:
Yes. Makes sense to me. Thanks!